### DISCUSSION TITLE
Popover not showing correctly for vue

### DISCUSSION THREAD

**Discussion by peterjslater** [tailwindlabs/headlessui#2006] [Help]
Hi
The popover is not showing correctly for vue but does work correctly for react.
You can see this on the headlessui website
https://headlessui.com/vue/popover
For React you can see the popover is displayed first time and clicking hides it.
For Vue (change the dropdown from Rect to Vue) the popover is not displayed and it takes two clicks to show it.

Also just noticed, if I click anywhere in the iframe and then click on the popover it also works, so looks like its a focus issue.

We are having the same problem in our app (also in an iframe like in the headlessui site) using latest version of headless (1.7.4) using Chrome (107.0.5304.88) on windows 11

I would hope a single click should activate the popup even in an iframe?

Is there a workaround, or should I create a bur report issue ?

**Comment by bradmanning** [comment]
I am having the same two-click/focus issue as well except in my scenario the Popover component is wrapped into a web component instead of being in an iFrame. If I place the same component containing the Popover directly in a mounted Vue instance this issue does not occur.

For reference, I am using HeadlessUI 1.7.4 (containing #1914) on Chrome (107.0.5304.106) Windows 10. I am happy to provide more info if needed.
